### PR TITLE
update SPEC_CONFORMANCE.md and add missing cases in corresponding tests

### LINKED ISSUE #145: Check new SPEC-CONFORMANCE.md lines and complete them if tests already exists

### PR DESCRIPTION
**Pull request by gilbrechbuhler**
Fix #145

### CODE REVIEW

File: testing/SPEC-CONFORMANCE.md [hunk 1]
```diff
@@ -77,8 +77,8 @@ The conformance requirements for EBU-TT Part 3 derive from the specification its
...
-|R71|3|If `timebase="smpte"` then the attributes `ttp:markerMode`, `ttp:frameRate` and `ttp:dropMode` SHALL be specified on `tt:tt`||
+|R70|3|If `ttp:timeBase="media"`` then the time expression of `begin` and `end` attributes SHALL have one of the following formats: <ul><li>hh:mm:ss followed by an optional decimal fraction (full-clock value). The number of hours SHALL NOT be restricted.</li><li>non-negative number followed by an optional decimal fraction followed by one of the symbols h, m, s, ms (time-count value)</li></ul>|`bdd/features/validation/timeBase\_timeformat\_constraints.feature`|
+|R71|3|If `timebase="smpte"` then the attributes `ttp:markerMode`, `ttp:frameRate` and `ttp:dropMode` SHALL be specified on `tt:tt`|`bdd/features/validation/smpte\_constraints.feature`|
 |R72|3|If `timebase="smpte"` then `ttp:markerMode` SHALL be specified and shall have the value `discontinuous`.||
```
Comment (gilbrechbuhler) [inline testing/SPEC-CONFORMANCE.md]: This one has become wrong by the way, we allow "continuous" now

Comment (nigelmegitt) [inline testing/SPEC-CONFORMANCE.md]: Agreed. Please could you raise an issue and a fix?

Comment (gilbrechbuhler) [inline testing/SPEC-CONFORMANCE.md]: May I fix the issue here, as this pull request already modifies this document ? Referencing it in the commit message of course.

Comment (nigelmegitt) [inline testing/SPEC-CONFORMANCE.md]: OK

**Comment by nigelmegitt** [comment]
Looks good to me, thanks Gil. About to merge.

**Comment by kozmaz87** [comment]
ACK

**Pull request merged**